docker/Dockerfile_base-jammy

```diff
@@ -0,0 +1,148 @@
+		gosu \
+		lcov \
+    libelf-dev \
+		libexpat-dev \
+    libvecmath-java \
+		libfreetype6-dev \
+		libgmp-dev \
+		libgtest-dev \
+		libisl-dev \
+		libmpc-dev \
+		libmpfr-dev \
+		libpng-dev \
+		libssl-dev \
+		lsb-release \
+		make \
+		ninja-build \
+		openssh-client \
+    openjdk-11-jre \
+    openjdk-11-jdk \
+		python3-dev \
+		python3-pip \
+		rsync \
+    screen \
+		shellcheck \
+		tzdata \
+    texinfo \
+    u-boot-tools \
+    util-linux \
+		unzip \
+		valgrind \
+		wget \
+		xsltproc \
+		zip \
+	&& apt-get -y autoremove \
+	&& apt-get clean autoclean \
+	&& rm -rf /var/lib/apt/lists/{apt,dpkg,cache,log} /tmp/* /var/tmp/*
+
+# Set java 11 as default
+sudo update-alternatives --set java $(update-alternatives --list java | grep "java-11")
+
+# gtest
+RUN cd /usr/src/gtest \
+	&& mkdir build && cd build \
+	&& cmake .. && make -j$(nproc) \
+	&& find . -name \*.a -exec cp {} /usr/lib \; \
+	&& cd .. && rm -rf build
+
+# Install Python 3 pip build dependencies first.
+RUN python3 -m pip install --upgrade pip wheel setuptools
+
+# Python 3 dependencies installed by pip
+RUN python3 -m pip install argparse argcomplete coverage cerberus empy jinja2 kconfiglib \
+		matplotlib==3.0.* numpy nunavut>=1.1.0 packaging pkgconfig pyros-genmsg pyulog \
+		pyyaml requests serial six toml psutil pyulog wheel jsonschema pynacl
+
+# manual ccache setup
+RUN ln -s /usr/bin/ccache /usr/lib/ccache/cc \
+	&& ln -s /usr/bin/ccache /usr/lib/ccache/c++
+
+# astyle v3.1
+RUN wget -q https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.1/astyle_3.1_linux.tar.gz -O /tmp/astyle.tar.gz \
+	&& cd /tmp && tar zxf astyle.tar.gz && cd astyle/src \
+	&& make -f ../build/gcc/Makefile -j$(nproc) && cp bin/astyle /usr/local/bin \
+	&& rm -rf /tmp/*
+
+# Gradle (Required to build Fast-RTPS-Gen)
+#RUN wget -q "https://services.gradle.org/distributions/gradle-6.3-rc-4-bin.zip" -O /tmp/gradle-6.3-rc-4-bin.zip \
+	#&& mkdir /opt/gradle \
+	#&& cd /tmp \
+	#&& unzip -d /opt/gradle gradle-6.3-rc-4-bin.zip \
+	#&& rm -rf /tmp/*
+
+#ENV PATH "/opt/gradle/gradle-6.3-rc-4/bin:$PATH"
+
+# Intall foonathan_memory from source as it is required to Fast-RTPS >= 1.9
+#RUN git clone https://github.com/eProsima/foonathan_memory_vendor.git /tmp/foonathan_memory \
+	#&& cd /tmp/foonathan_memory \
+	#&& mkdir build && cd build \
+	#&& cmake .. \
+	#&& cmake --build . --target install -- -j $(nproc) \
+	#&& rm -rf /tmp/*
+
+# Fast-DDS (Fast-RTPS 2.0.2)
+#RUN git clone --recursive https://github.com/eProsima/Fast-DDS.git -b v2.0.2 /tmp/FastDDS-2.0.2 \
+	#&& cd /tmp/FastDDS-2.0.2 \
+	#&& mkdir build && cd build \
+	#&& cmake -DTHIRDPARTY=ON -DSECURITY=ON .. \
+	#&& cmake --build . --target install -- -j $(nproc) \
+	#&& rm -rf /tmp/*
+
+# Fast-RTPS-Gen 1.0.4
+#RUN git clone --recursive https://github.com/eProsima/Fast-DDS-Gen.git -b v1.0.4 /tmp/Fast-RTPS-Gen-1.0.4 \
+	#&& cd /tmp/Fast-RTPS-Gen-1.0.4 \
+	#&& gradle assemble \
+	#&& gradle install \
+	#&& rm -rf /tmp/*
```

Comment:
These dependencies are need for the `_rtps` targets. Do we want to use this container for PX4 version `1.13`?
Otherwise they can be removed as `v1.14` does not need them.

Reply:
I think we don't officially support 22.04 in 1.13, although it can be used i think we can use the opportunity to simplify 

I'll remove this